styles/style.css

```diff
@@ -0,0 +1,201 @@
+/* CSS Startly */
+   
+nav ul {
+  display: grid;
+  grid-template-columns: 1fr 5fr 1fr 1fr;
+  list-style-type: none;
+  text-align: right;
+  padding-top: 10px;
+  padding-bottom: 30px;
+  font-weight: bold;
+}
+
+nav ul li:hover {
+  color: turquoise;
+}
+
+body{
+  font-family: 'PT Sans', sans-serif;;
+}
+
+h1{
+  font-size: 40px;
+}
+
+.welcome {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: repeat(2, 75px);
+  text-align: center;
+  padding-bottom: 40px;
+  background-color: turquoise;
+}
+
+.tagline {
+  padding-top: 20px;
+  font-size: 25px;
+}
+
+a[href="#"]{
+  display: inline-block;
+  background-color: white; 
+  border: 2px solid lightgrey;
+  border-radius: 10px; 
+  color: black;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 25px;
+  width: 250px;
+  margin-left:auto;
+  margin-right:auto;
+}
+
+.quote1{
```

Comment:
The characters overlap here if the window is made smaller. What do you think would happen if you used flexbox?